bridge-circuit-host/src/bridge_circuit_host.rs

```diff
@@ -80,24 +88,25 @@ pub fn prove_bridge_circuit(
     //     panic!("Light client proof receipt verification failed");
     // }
 
-    // Check for number of watchtowers
```

Comment:
can we instead add verification that the watchtower input arrays have the same length?